docs/concepts/trading-on-vega/discounts-rewards.md

```diff
@@ -104,9 +104,11 @@ Some rewards measure trader activity over a number of epochs (set per reward). T
 ### How rewards are scaled
 Since rewards can only be provided if they're funded, the [recurring tranfer](../assets/transfers.md#recurring-transfers) that's used to fund those rewards also includes details on how the final reward amount is calculated.
 
-**Pro-rata**: A participant's reward is scaled based on their activity streak and/or how long they've kept previous reward earnings in their vested rewards account, which will influence how high up the rankings they are.
+**Pro-rata**: A participant's reward is scaled based on their score.
 
-**Rank**: A participant's reward is scaled based on where they sit on the list of traders who are eligible for the reward. Those higher up the list receive a higher ratio of the reward for each reward period.
+**Rank**: A participant's reward is scaled based on where their score sits on the list of traders who are eligible for the reward. Those higher up the list receive a higher ratio of the reward for each reward period.
+
```

Comment:
Would be very unusual for the community to propose such a recurring transfer, but technically the rank table could allow worse performing users to receive a higher share of the rewards.

Would some comment saying look at the rank table in the transfers API be better here? Similar to what we have in other places the API is mentioned.